includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1740,12 +1740,14 @@ private function finalize_styles() {
 			$this->amp_custom_style_element->appendChild( $this->dom->createTextNode( $css ) );
 
 			$included_size    = 0;
+			$excluded_size    = 0;
 			$included_sources = array();
 			foreach ( $stylesheet_sets['custom']['pending_stylesheets'] as $i => $pending_stylesheet ) {
 				if ( ! ( $pending_stylesheet['node'] instanceof DOMElement ) ) {
 					continue;
 				}
-				$message = $pending_stylesheet['node']->nodeName;
+				$message  = sprintf( '% 6dB: ', $pending_stylesheet['size'] );
```

Comment:
Could a space between the number and the `B` be added to make it easier to read?